include/odp/api/spec/schedule.h

```diff
@@ -223,10 +223,14 @@ void odp_schedule_prefetch(int num);
 /**
  * Maximum scheduling priority level
  *
- * This is the maximum value that can be set to 'prio' field in
- * odp_schedule_param_t (e.g. odp_queue_create()). Queues with a higher
- * priority value are served with higher priority than queues with a lower
- * priority value.
+ * This is the global maximum value that can be set to 'prio' field in
+ * odp_schedule_param_t (e.g. odp_queue_create()). Configured based on
+ * odp_schedule_config_t::num_prios. Queues with a higher priority value are
```

Comment:
odp_schedule_config_t does not have num_prios anymore. Could be now: "Configured based on odp_schedule_config_t::prio... ", since prio.min / prio.num would not create correct Doxygen linking (?).